crates/pessimistic-proof-test-suite/src/runner.rs

```diff
@@ -41,8 +42,13 @@ impl Runner {
     }
 
     /// Extract outputs from the committed public values.
-    pub fn extract_output(mut public_vals: SP1PublicValues) -> PessimisticProofOutput {
-        public_vals.read::<PessimisticProofOutput>()
+    pub fn extract_output(public_vals: SP1PublicValues) -> PessimisticProofOutput {
+        bincode::DefaultOptions::new()
+            .with_big_endian()
+            .with_fixint_encoding()
+            .allow_trailing_bytes()
```

Comment:
Ideally these options should be taken out into a helper function so they can be easily shared between the encoding and decoding side for consistency.